### PR TITLE
News5

### PR DESCRIPTION
**Pull request by Castelli87**
Done the ticket 5 but i have problem to merge the ticket 4 in the main

### CODE REVIEW

File: src/Components/CommentsList.jsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+function CommentsList() {
```
Comment (HannahBurks) [inline src/Components/CommentsList.jsx]: We can remove this component for now as it is not relevant to the current ticket (5) - we can instead include it is the next ticket pr :)

File: src/Components/ArticleMainCard.jsx [hunk 2]
```diff
@@ -1,9 +1,51 @@
...
+      </p>
+
+      <p>{currentArticle.body}</p>
+      <p>
+        <button>👍</button> {currentArticle.votes}
+      </p>
+      <p>{currentArticle.created_at}</p>
+      <form>
+        <label>
+          Comment here:
```
Comment (HannahBurks) [inline src/Components/ArticleMainCard.jsx]: I would remove the comment section for now as it is not relevant to this ticket, but instead the next one. :)

File: src/Api/api.js [hunk 3]
```diff
@@ -10,3 +10,10 @@ return ncNews
 .get(`/articles`)
 .then(({data})=> data)}
 
+ export const fetchArticle = (article_id)=>{
+
+    return ncNews
+    .get(`/articles/${article_id}`)
+    .then((resp)=>resp.data.article)
+    .catch((err)=>console.log(err))
```
Comment (HannahBurks) [inline src/Api/api.js]: We can remove the .catch here as any errors from the api will be coming back as resp but will have a key of message or similar (as this was handled in the back end). For errors on the front end, we need to instead always check the response to see if it is an error message or the data. :)

**Review by HannahBurks — CHANGES_REQUESTED** [review]
Hey Davide, nice work. I would just remove anything comment or voting related for now as it is not relevant to this ticket. Please also remove your catch block. Once amended, please send another PR and you should be good to go! 😄

File: src/Components/ArticleMainCard.jsx (hunk 2 above)
Comment (Castelli87) [reply src/Components/ArticleMainCard.jsx]: Thanks Hannah for your feedback i gonna remove this part of code.Anyway i wrote this code like a "template" with not functionality to stick with my plan so i can see where are the stuff that i need to implement later on.

File: src/Components/CommentsList.jsx (hunk 1 above)
Comment (Castelli87) [reply src/Components/CommentsList.jsx]: Thanks Hannah for your feedback i gonna remove this part of code.Anyway i wrote this code like a "template" with not functionality to stick with my plan so i can see where are the stuff that i need to implement later on.

File: src/Api/api.js (hunk 3 above)
Comment (Castelli87) [reply src/Api/api.js]: Ok perfect thank you for your feedback and for the help

**Review by dappernerddesigns — APPROVED** [review]
Nicely done Davide, merge away :)

**Pull request merged**